### PR TITLE
perf(route): 添加过滤，减少请求时间

### PR DESCRIPTION
**Pull request by hualiong**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/sicau/jk/0/0/2/8d95466cf63e537292b303cb92b5958c
/sicau/jk/53/11/1/8d95466cf63e537292b303cb92b5958c
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

文档排版以及添加过滤，减少请求时间

### CODE REVIEW

File: lib/routes/sicau/jk.ts [hunk 1]
```diff
@@ -128,27 +82,31 @@ export const route: Route = {
...
+                query: {
+                    gid: gidDict[gid],
+                    typeId: typeDict[typeId],
+                    sortType,
+                    page,
+                },
+            });
+
+        const res1 = await query(1);
+        const res2 = await query(2);
```
Comment (TonyRL) [inline lib/routes/sicau/jk.ts:96]: Please parse only page 1.

Comment (hualiong) [reply lib/routes/sicau/jk.ts:96]: There is not enough data on the page 1, it doesn't cover all the newly released activities.

Comment (TonyRL) [reply lib/routes/sicau/jk.ts:96]: Please perform the requests in parallel with `Promise.all()` instead of in series.

Comment (hualiong) [reply lib/routes/sicau/jk.ts]: OK

**Pull request merged**